src/main/java/frc/trigon/robot/constants/CameraConstants.java

```diff
@@ -35,6 +35,7 @@ public class CameraConstants {
     private static final double
             AVERAGE_PIXEL_ERROR = 0.25,
             PIXEL_STANDARD_DEVIATIONS = 0.08;
+    public static final boolean SHOULD_USE_CAMERA_SIMULATION = true;
```

Comment:
Can't you just use `RobotHardwareStats.isSimulation()` ?

Reply:
We don't always want to use it. It takes a lot of processing power